bot.py

```diff
@@ -467,6 +573,26 @@ async def show_queue(self, message, _command_content):
 
         await message.channel.send(reply)
 
+    async def show_help(self, message, command_content):
+        """
+        Show link to full documentation
+        """
+        reply = "```\n"
+        if len(command_content) > 0:
+            reply += self.help_messages[command_content]
+        else:
+            for help_message in self.help_messages.values():
```

Comment:
Optional suggestion: show commands in sorted command name order.
```suggestion
            for _, help_message in sorted(self.help_messages.items()):
```